src/pages/lib/auth/manage-session/q/platform/[platform].mdx

```diff
@@ -0,0 +1,204 @@
+export const meta = {
+  title: `Manage user session and credentials`,
+  description: `Learn how to manage user sessions and credentials.`,
+};
+
```

Comment:
General note @abdallahshaban557 this updated page does not include this section on security tokens -  https://docs.amplify.aws/lib/auth/manageusers/q/platform/js/#managing-security-tokens - should this be added to this content or moved to another section.

Reply:
I always found this super confusing to display in our documentation. Let us keep it out for now.

Reply:
if we get requests to add it back -we can

Reply:
Ack added it to my notes on used/unused fragments.